core/storage/ipfs/api_ipfs_datastore/api_ipfs_datastore.cpp

```diff
@@ -13,9 +13,6 @@ namespace fc::storage::ipfs {
   outcome::result<bool> ApiIpfsDatastore::contains(const CID &key) const {
     auto res = api_->ChainReadObj(key);
     if (res.has_error()) {
-      if (res.error() == IpfsDatastoreError::kNotFound) {
```

Comment:
If 'kNotFound' is treated as error, why return value is 'bool'? It is always 'true' or 'error'.

Reply:
Rpc can't return `kNotFound`, so treating error consistently